manifests/server/puppetserver.pp

```diff
@@ -169,6 +169,8 @@
       context => '/files/etc/rc.conf',
       changes => ["set puppetserver_java_opts '\"${jvm_cmd}\"'"],
     }
+  } elsif $facts['os']['family'] == 'Debian' and !$puppet::params::aio_package {
+    $server_gem_paths = ['${jruby-puppet.gem-home}', '/usr/lib/puppetserver/vendored-jruby-gems'] # lint:ignore:single_quote_string_with_variables
```

Comment:
Given the repetition in declaring a vendored-jruby-gems dir, is this something that belongs in `params.pp`?

Reply:
Declaring `$server_gem_paths` in `puppet::params` was my first idea too, however it doesn't work because the variable isn't a class parameter of `puppet::server::puppetserver`.